conf/jasminegraph-server.properties

```diff
@@ -58,6 +58,7 @@ org.jasminegraph.fl.weights.file=/var/tmp/jasminegraph-localstore/weights.txt
 org.jasminegraph.fl.flag.file=/var/tmp/jasminegraph-localstore/flag.txt
 org.jasminegraph.fl.aggregator=true
 org.jasminegraph.fl.org.training=true
+org.jasminegraph.fl.organization.file=/var/tmp/jasminegraph-localstore/orgs_details.txt
```

Comment:
`orgs_details.txt` file is a server configuration file. It should be located in the `conf` folder of JasmineGraph.

Reply:
updated in c9a8c3488457482c9d93f024346727a896ea2015

Reply:
Done